packages/analytics/src/analytics.ts

```diff
@@ -1,6 +1,28 @@
 import mixpanel, { Mixpanel } from 'mixpanel';
 import R from 'ramda';
 
+function serialize(doc: Record<string, unknown>) {
+  const obj: Record<string, unknown> = doc;
+
+  return R.reduce(
+    (accum, [field, value]) => {
+      if (value instanceof Buffer) {
+        return {
+          ...accum,
+          [field]: (value as Buffer).toString('hex'),
+        };
+      }
+
```

Comment:
We should also support serializing instance of `ObjectId`. You can refer to this:
https://github.com/HighOutputVentures/highoutput-library/blob/main/packages/repository/src/repository.ts#L76-L84